cmake/platform/types/PlatformTypes.h

```diff
@@ -47,8 +47,11 @@ typedef unsigned int PlatformUIntType;
 typedef PlatformIntType PlatformIndexType;
 #define PRI_PlatformIndexType PRI_PlatformIntType
 
-typedef PlatformUIntType PlatformSizeType;
-#define PRI_PlatformSizeType PRI_PlatformUIntType
+typedef int64_t PlatformSignedSizeType;
```

Comment:
Shouldn't this be ssize_t?

Reply:
I chose to use `int64_t` instead of `ssize_t` as this is a fixed-size integer and would remove the warnings about "use of un-sized integers".   This file is only for supporting the core systems Linux, Arm/Linux, and Darwin.  To the best of my knowledge each system in that list support 64bit.